apps/volontulo/frontend/scss/customization/_customization.scss

```diff
@@ -360,9 +360,15 @@ input.form-control[required] {
         }
     }
     .user-photo {
-        img {
-            width: 100%;
-            max-height: 300px;
+        .cover-image {
+            height: 250px;
+            background-repeat: no-repeat;
```

Comment:
@magul This has been written after quite a lot of beer on Friday ;). Please better check those styles before potential merge ;)